nicegui/templates/index.html

```diff
@@ -70,12 +70,24 @@
           props[event_name] = handler;
         });
         const slots = {};
-        Object.entries(element.slots).forEach(([name, ids]) => {
-          const children = ids.map(id => renderRecursively(elements, id));
-          if (name == 'default' && element.text) {
-            children.unshift(element.text);
+        Object.entries(element.slots).forEach(([name, data]) => {
+          slots[name] = (props) => {
+            const rendered = [];
+            if (data.template) {
+              rendered.push(Vue.h({
+                props: { props: { type: Object, default: {} } },
+                template: data.template,
+              }, {
+                props: props
+              }));
+            }
+            const ids = ('ids' in data) ? data.ids : data;
```

Comment:
Do we really need the second case where `ids = data`? Shouldn't there always be `data.ids`?

Reply:
Nice catch ! Indeed, line 84 is no more needed. I add many back and forth on this trying things and that was my first attempt when slots and scoped_slots where in separated structure. Anyway: here is done a commit to clean that.